src/Sfa.Tl.ResultsAndCertification.Database/Tables/OverallResult.sql

```diff
@@ -4,6 +4,7 @@
 	[TqRegistrationPathwayId] INT NOT NULL,
 	[Details] NVARCHAR(MAX) NULL,
 	[ResultAwarded] NVARCHAR(50),
+	[SpecialismResultAwarded] NVARCHAR(50) NULL,
```

Comment:
Is SpecialismResultAwarded column added in Model OverallResult.cs ?

Reply:
Yes it added to the model already